src/QA/QA.Web/ViewModels/ReviewViewModel.cs

```diff
@@ -31,7 +31,11 @@ public class
         public string OutcomeDescription { get; internal set; }
         public string HoursPerWeek { get; internal set; }
         public string PossibleStartDate { get; internal set; }
+        public string ProviderContactName { get; internal set; }
+        public string ProviderContactEmail { get; internal set; }
+        public string ProviderContactTelephone { get; internal set; }
         public string ProviderName { get; internal set; }
+        public SourceOrigin SourceOrigin { get; internal set; }
```

Comment:
I don't think we need the SourceOrigin on this ViewModel to work out `IsEmployerVacancy`, could you please use the `OwnerType` property from the vacancy instead please?